frontend/src/views/Collection/index.tsx

```diff
@@ -59,6 +60,9 @@ const Collection: FC<Props> = ({ id = "" }) => {
       <CollectionInfo>
         <H3>{collection.name}</H3>
         <Description>{collection.description}</Description>
+        <ContactContainer>
+          {renderContact(collection.contact_name, collection.contact_email)}
+        </ContactContainer>
         <LinkContainer>{renderLinks(collection.links)}</LinkContainer>
```

Comment:
```suggestion
        <LinkContainer>
          {renderLinks(collection.links)}
          {renderContact(collection.contact_name, collection.contact_email)}
        </LinkContainer>
```

The contacts just need to be put into the grid container, no need to change the name of the container since the contact is semantically a link

Reply:
will it add it as a `mailto:` link